apps/src/blockly/googleBlocklyWrapper.ts

```diff
@@ -621,10 +621,6 @@ function initializeBlocklyWrapper(blocklyInstance: GoogleBlocklyInstance) {
     );
   };
 
-  extendedWorkspaceSvg.isReadOnly = function () {
-    return blocklyWrapper.readOnly || this.options.readOnly;
-  };
-
```

Comment:
See https://codedotorg.slack.com/archives/C04TH8400AU/p1750957867971139. This was removed to fix the behavior of the `workspace.setIsReadOnly()` function